src/DeserializationPojo/ListUsersPojo.java

```diff
@@ -0,0 +1,58 @@
+package DeserializationPojo;
+
+import java.util.List;
+
+
+
+public class ListUsersPojo {
```

Comment:
Now this represents an object that is returned only by list users. So we can name it `ListUsersResponsePojo`. The word Response makes it much more obvious what the class is.